config/index.js

```diff
@@ -436,42 +436,6 @@ var conf = convict({
     env: 'CLIENT_ADDRESS_DEPTH',
     default: 3
   },
-  signinConfirmation: {
-    enabled: {
-      doc: 'enable signin confirmation',
-      format: Boolean,
-      default: false,
-      env: 'SIGNIN_CONFIRMATION_ENABLED'
-    },
-    sample_rate: {
-      doc: 'signin confirmation sample rate, between 0.0 and 1.0',
-      format: Number,
-      default: 1.0,
-      env: 'SIGNIN_CONFIRMATION_RATE'
-    },
-    supportedClients: {
-      doc: 'support sign-in confirmation for only these clients',
-      format: Array,
-      default: [
-        'iframe',
-        'fx_firstrun_v1',
-        'fx_firstrun_v2',
-        'fx_desktop_v1',
-        'fx_desktop_v2',
-        'fx_desktop_v3',
-        'fx_ios_v1',
-        'fx_ios_v2',
-        'fx_fennec_v1'
-      ],
-      env: 'SIGNIN_CONFIRMATION_SUPPORTED_CLIENTS'
-    },
-    enabledEmailAddresses: {
```

Comment:
I'm glad this hasn't merged. I think we should keep `enabledEmailAddresses` for functional test purposes. See [the reasoning here](https://github.com/mozilla/fxa-content-server/issues/4401#issuecomment-260435778).